build.sh

```diff
@@ -69,6 +69,19 @@ initDistroRid()
     fi
 }
 
+initTargetDistroRid()
```

Comment:
Given the above, I would suggest that you rename the current $__DistroRid to $__HostDistroRid and your proposed $__CrossDistroRid as $__DistroRid. This will maintain the existing uses and convey the correct meaning via $__DistroRid and also eliminate your changes in dir.props.